ci-operator/step-registry/openshift-qe/installer/azure/openshift-qe-installer-azure-workflow.metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+	"path": "openshift-qe/installer/azure/openshift-qe-installer-azure-workflow.yaml",
```

Comment:
Let's create this file using a symbolic link to the OWNERS file that's in the OpenShift-qe folder and add Ravi there instead